app/web/src/store/feature_flags.store.ts

```diff
@@ -3,23 +3,11 @@ import { addStoreHooks } from "@si/vue-lib/pinia";
 import * as _ from "lodash-es";
 import { posthog } from "@/utils/posthog";
 
-type FlagStoreKey =
-  | "SINGLE_MODEL_SCREEN"
-  | "MODULES_TAB"
-  | "CONTRIBUTE_BUTTON"
-  | "SECRETS";
+type FlagStoreKey = "CONTRIBUTE_BUTTON" | "SECRETS";
 
 // The key to this object is the flag name on PostHog, the "storeKey" property is the
 // key used when accessing it from the Pinia feature flag store
 const FLAGS: { [key: string]: { storeKey: FlagStoreKey; default: boolean } } = {
-  one_screen_to_rule_them_all: {
-    storeKey: "SINGLE_MODEL_SCREEN",
-    default: false,
-  },
-  modules_tab: {
```

Comment:
We want the modules tab flag to still be in here

Reply:
Do we? Isn't that enabled for all and stable?

I thought we only wanted to keep the contribute button and the secrets. I can roll it back.